### ISSUE TITLE
Block the login dialog

### ISSUE THREAD

**Issue by hbcarlos** [davidbrochart/jupyterlab-auth#6]
The logging dialog is blocked, so users can't skip it without adding a name or logging in with a third-party auth method. The problem is that in some browsers, when clicking outside the dialog, it closes. Another issue, we don't check if the user introduced an empty name.

**Comment by davidbrochart** [comment]
Anyway, the logging dialog probably appears "too late" currently. We should not even be able to see the workspace if we are not logged in, right?

**Comment by hbcarlos** [comment]
Yes. I think would be best to change to a login page before jupyterLab loads
